src/pages/blog/proactive-data-security.pug

```diff
@@ -0,0 +1,107 @@
+extends ../../layouts/content.pug
+
+block prepend vars
+  -var pageTitle = "A proactive approach to data security | Leaf"
+  -var metaUrl = "https://weareleaf.com/blog/proactive-data-security"
+  -var metaTitle = "A proactive approach to data security"
+  -var metaDescription = "Learn how focusing on data security can benefit both your development cycle and your business."
+
+  -var heroHeading = "A proactive approach to data security"
+  -var heroSubheading = "(by Tina van Schelt)"
+
+block pageComponents
+
+  +banner({
+  })
+
+  .banner-spacer
+
+  +hero({
+    heroHeading: heroHeading,
+    heroSubheading: heroSubheading,
+  })
+
+  .section
+    .container
+
+      .grid.grid--center
+
+        +share-icons({
+          socialModifier: "inline"
+        })
+
+        .blog-post
+
+          p.blog-post__intro If you take a look at the general trend of the current online dialogue surrounding software development you’ll, rightfully, find a big emphasis on performance and accessibility, but not so much on their less trendy cousin, security.
+
+          p.blog-post__text According to a <a href="https://about.gitlab.com/blog/2019/07/15/global-developer-report/" target="_blank">2019 Global Developer Report</a> in which GitLab surveyed 4,071 individuals, <i>69% of developers indicate they’re expected to write secure code, nearly half of security professional surveyed (49%) said they struggle to get developers to make remediation of vulnerabilities a priority and </i>
```

Comment:
What made you go for the `<i>` element over the `<em>`? I've read https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i, and I'm not still entirely clear, but this exactly _feels_ to be like it falls into the `<em>` definition, whereas a few paragraphs down, the `<i>https</i>` feels better suited to `<i>`.

All of that said, happy to be schooled.